src/month.js

```diff
@@ -60,10 +67,18 @@ export default function CalendarMonth(props) {
   const numberOfWeeks = _.size(dayWeeks);
 
   return (
-    <div className={className}>
-      <h3 className={headerClassName}>
-        {firstDay.format(headerFormat)}
-      </h3>
+    <div
+      className={classNames(
+        className,
+        { 'tt-cal-headerInsideDay': headerInsideDay }
+      )}
+    >
+      { headerInsideDay ?
```

Comment:
nit: isn't our convention to include "inside" spaces for object declarations, but _not_ with injected javascript e.g. ternary expressions?